tests/identifiers/test_datacite.py

```diff
@@ -0,0 +1,189 @@
+# -*- coding: utf-8 -*-
+import os
+import mock
+import lxml
+import pytest
+import responses
+from nose.tools import *  # noqa
+
+from datacite import schema40
+
+from website import settings
+from website.app import init_addons
+from website.identifiers.clients import DataCiteClient
+from website.identifiers import metadata
+
+from tests.base import OsfTestCase
+from tests.identifiers.conftest import datacite_metadata_response
+from tests.test_addons import assert_urls_equal
+from osf_tests.factories import AuthUserFactory, RegistrationFactory
+
+class MockDataciteClient(object):
+
+    def __init__(self, *arg, **kwargs):
+        pass
+
+    metadata_get = mock.Mock(return_value=datacite_metadata_response())
+    metadata_post = mock.Mock(return_value='OK (10.5072/FK2osf.io/yvzp4)')
+
+@pytest.fixture()
+def datacite_client():
+    return DataCiteClient()
+
+@pytest.fixture()
+def registration():
+    return RegistrationFactory()
+
+@pytest.mark.django_db
+class TestDataCiteClient:
+
+    @responses.activate
+    @mock.patch('website.identifiers.clients.datacite.DataCiteMDSClient', MockDataciteClient)
+    @mock.patch('website.settings.DATACITE_URL', 'https://mds.fake.datacite.org')
+    def test_datacite_create_identifiers(self, datacite_client, datacite_node_metadata):
+        responses.add(
```

Comment:
Why is it necessary to mock out these requests if you're already mocking out `DataCiteMDSClient`?